### PR TITLE
add form param attachment

### PR DESCRIPTION
**Pull request by Phoenix124**
#### Checklist
- [x] [Sign Allure CLA][cla]
- [x] Provide unit tests

[cla]: https://cla-assistant.io/accept/allure-framework/allure2

### CODE REVIEW

**Comment by Phoenix124** [comment]
@baev можете глянуть?
это для аттача formParam

File: allure-attachments/src/main/java/io/qameta/allure/attachment/http/HttpRequestAttachment.java [hunk 1]
```diff
@@ -41,16 +41,19 @@ public class HttpRequestAttachment implements AttachmentData {
 
     private final Map<String, String> cookies;
 
+    private final Map<String, String> formParams;
+
     public HttpRequestAttachment(final String name, final String url, final String method,
                                  final String body, final String curl, final Map<String, String> headers,
-                                 final Map<String, String> cookies) {
+                                 final Map<String, String> cookies, final Map<String, String> formParams) {
```
Comment (baev) [inline allure-attachments/src/main/java/io/qameta/allure/attachment/http/HttpRequestAttachment.java:48]: Let's keep the old constructor and create a new one to make the change backwards-compatible.

**Review by baev — CHANGES_REQUESTED** [review]

File: allure-attachments/src/main/java/io/qameta/allure/attachment/http/HttpRequestAttachment.java (hunk 1 above)
Comment (Phoenix124) [reply allure-attachments/src/main/java/io/qameta/allure/attachment/http/HttpRequestAttachment.java]: fixed

File: allure-attachments/src/main/java/io/qameta/allure/attachment/http/HttpRequestAttachment.java [hunk 2]
```diff
@@ -53,6 +55,17 @@ public HttpRequestAttachment(final String name, final String url, final String m
         this.cookies = cookies;
     }
 
+    public HttpRequestAttachment(final Builder builder, final Map<String, String> formParams) {
```
Comment (baev) [inline allure-attachments/src/main/java/io/qameta/allure/attachment/http/HttpRequestAttachment.java:58]: The old constructor should be calling the new one (providing `null` as a last argument)

**Review by baev — CHANGES_REQUESTED** [review]
please fix issues from the comment from CLA assistant

**Pull request closed without merging**